master/Dockerfile.ubuntu

```diff
@@ -10,19 +12,29 @@ MAINTAINER  Buildbot maintainers
 # that everything is rebuilt
 ENV         security_updates_as_of 2016-07-30
 
+# This will make apt-get install without question
+ENV         DEBIAN_FRONTEND noninteractive
```

Comment:
Thanks, I wasn't aware of that.

but... I disagree. The only argument is that when you do docker exec -it bash, then you wont have interactive debian stuff anymore.

Those containers are not meant for interactive (who does?), and I think it is better to put that environment rather than force setting it inside each RUN command

There are some packages that wont install if you dont put this variable as they insist to ask question. This is just lost time to try and debug those.

the discussion is from 2014. they may have change their mind since, and they dont mention it in https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/